libs/diagnostics/performance_handler.hpp

```diff
@@ -76,18 +69,18 @@ struct Recorder {
   // Record to a histogram safely across threads. Please use this method sparingly. It should not be necessary in most
   // cases. Do NOT mix calls with `record` in the same recorder.
   void recordAtomic(int64_t val);
-
-  hdr_interval_recorder recorder;
+  histogram<axes_t> recorder;
```

Comment:
can the recorder be renamed to `histogram_` because I'm confused with the outer class name

Reply:
correct, I have renamed it.